src/main/java/seedu/address/logic/parser/ArgumentTokenizer.java

```diff
@@ -62,7 +62,7 @@ public static ArgumentMultimap tokenize(String argsString, Prefix... prefixes) {
      * {@code argsString} starting from index {@code fromIndex}. An occurrence
      * is valid if there is a whitespace before {@code prefix}. Returns -1 if no
      * such occurrence can be found.
-     *
+     * <p>
```

Comment:
Maybe someone can help me review this because I'm not sure if this is common practice or not @tonishka @ckcherry23

Reply:
`<p>` is used within the AB3 code and is also seen in the textbook so should be fine.

<img width="698" alt="Screenshot 2022-03-10 at 2 26 07 PM" src="https://user-images.githubusercontent.com/68203159/157602138-27cbcf1f-2066-4f03-9ac5-ed494445642f.png">
.